script/track_commission_changes.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
```

Comment:
What is the purpose of this script?
I don't see anything about commission changes here (I'm refering to the script name).
Also, if it meant for dev environment, then let's move it to the dev directory.